lib/grpc/server/adapters/cowboy.ex

```diff
@@ -42,13 +42,26 @@ defmodule GRPC.Server.Adapters.Cowboy do
         {:ranch_tcp, :cowboy_clear}
       end
 
-    {ref, mfa, type, timeout, kind, modules} =
-      :ranch.child_spec(ref, transport, trans_opts, protocol, proto_opts)
+    case :ranch.child_spec(ref, transport, trans_opts, protocol, proto_opts) do
```

Comment:
If we update ranch, we won't need this `case`

Reply:
If we update `:ranch`, then the tuple version would never match afterward, the dialyzer would see it as warning

Reply:
Yeah, because we won't need to support the tuple version :)

Reply:
we can't update ranch because cowlib still dependent on version 1.8.0 :(